yjit/src/cruby.rs

```diff
@@ -129,6 +134,12 @@ extern "C" {
 
     // Ruby only defines this in vm_insnhelper.c, which bindgen has trouble using b/c of duplicate definitions
     pub fn rb_vm_opt_mod(recv:VALUE, obj: VALUE) -> VALUE;
+
+    #[link_name = "rb_yarv_str_eql_internal"]
```

Comment:
I try to avoid that in general. It's weird for these bindings, because you wind up with a C binding wrapping a static inline function with the same name as the function wrapping it. There might be some way to make that work? But if you try it the obvious way, you get a wrapper where the wrapped function has the same name as the outer function you're trying to export.